src/components/SuperDrawer/SuperDrawer.tsx

```diff
@@ -53,16 +53,11 @@ export function SuperDrawer(): ReactPortal | null {
 
   // Also get the first / non-detail element on the stack
   const firstContent = contentStack.current[0].opts as OpenInDrawerOpts;
-  const { onPrevClick, onNextClick, onClose, titleRightContent, titleLeftContent } = firstContent;
+  const { onPrevClick, onNextClick, titleRightContent, titleLeftContent } = firstContent;
 
   const isDetail = currentContent !== firstContent;
   const title = currentContent.title || firstContent.title;
 
-  function handleOnClose() {
-    if (onClose) return onClose();
-    return closeDrawer();
-  }
-
   return createPortal(
```

Comment:
I've tried to make a test for this but it required a lot of mocking (rendering the drawer, then after render create a check, then close) for what it's worth now that it's resolved.